src/protocol/h1/parser/mod.rs

```diff
@@ -133,6 +129,22 @@ fn process_headers<T: AsBuffer>(kawa: &mut Kawa<T>) {
             }
         }
     }
+    // If chunked, elide all `Content-Length`
+    if kawa.body_size == BodySize::Chunked {
+        for block in &mut kawa.blocks {
+            let Block::Header(header) = block else {
+                continue;
+            };
+            let Store::Slice(key) = &header.key else {
+                unreachable!()
+            };
+            let key = key.data(buf);
+            if compare_no_case(key, b"content-length") {
+                println!("WARNING: Found both a Transfer-Encoding and a Content-Length, ignoring the latter");
```

Comment:
println should be avoid and replace by the log crate to be able to be print using standard logger and sozu one

Reply:
There are other places in the crate where `println` is used. Shouldn't this be done in a separate PR?